packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleExpandable.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Avatar,
+  Box,
+  BoxProps,
+  BoxStylesProps,
+  Card,
+  cardFocusableBehavior,
+  ComponentVariablesInput,
+  compose,
+  Flex,
+  Image,
+  Provider,
+  Text,
+  ThemeInput,
+} from '@fluentui/react-northstar';
+import * as React from 'react';
+
+// Compose custom styled Box
+type ExpandableBoxProps = {
+  expand?: boolean;
+};
+
+type ExpandableBoxStylesProps = ExpandableBoxProps;
+
+const ExpandableBox = compose<'button', ExpandableBoxProps, ExpandableBoxStylesProps, BoxProps, {}>(Box, {
+  className: 'ui-expandable-box',
+  displayName: 'ExpandableBox',
+  mapPropsToStylesProps: props => ({ expand: props.expand }),
+  handledProps: ['expand'],
+});
```

Comment:
```suggestion
});

ExpandableBox.defualtProps = {
  as: 'button',
}
```

If it's really button